target/ppc/translate/storage-ctrl-impl.c.inc

```diff
@@ -77,6 +77,20 @@ static bool trans_SLBIAG(DisasContext *ctx, arg_SLBIAG *a)
     return true;
 }
 
+static bool trans_SLBMTE(DisasContext *ctx, arg_SLBMTE *a)
+{
+    REQUIRE_64BIT(ctx);
+    REQUIRE_INSNS_FLAGS(ctx, SEGMENT_64B);
+    REQUIRE_SV(ctx);
+
+#if !defined(CONFIG_USER_ONLY) && defined(TARGET_PPC64)
+    gen_helper_store_slb(cpu_env, cpu_gpr[a->rb], cpu_gpr[a->rt]);
```

Comment:
ditto

Reply:
Fixed in 5138bfaa4fc2a26abe700d5e5ee17e05466dbcdd